From 06b89592d7aff44b0c62eadf8f927e90cd528a35 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 16 Mar 2021 11:55:50 +0200
Subject: [PATCH] [SW-34382] preboot: greco_arc: increase sram

Handling boot info from scratchpad sram, specifically nvme
descriptor, has increased the bss utilization.
increasing to a max of 64kb

Change-Id: I13d1633688ba2b1dd5eac4a7cd2f8b1f8973bc6e
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/greco_arc/common.h b/greco_arc/common.h
index aaed039..1eafa6e 100644
--- a/greco_arc/common.h
+++ b/greco_arc/common.h
@@ -11,7 +11,7 @@
 #include <bitops.h>
 
 #define PPBOOT_SIZE		0x4000		/* 16KB */
-#define PREBOOT_SIZE		0xC000		/* 48KB */
+#define PREBOOT_SIZE		0x10000		/* 64KB */
 
 #define HL_FW_BOOT_PTRS_BASE		0x80000
 #define HL_SPI_LEGACY_PREBOOT_OFF	0xC0000
diff --git a/greco_arc_gnu/common.h b/greco_arc_gnu/common.h
index 61cd863..f069c99 100644
--- a/greco_arc_gnu/common.h
+++ b/greco_arc_gnu/common.h
@@ -11,7 +11,7 @@
 #define SRAM_SIZE			0x8000000	/* 128MB */
 
 #define PPBOOT_SIZE			0x4000		/* 16KB */
-#define PREBOOT_SIZE			0xC000		/* 48KB */
+#define PREBOOT_SIZE			0x10000		/* 64KB */
 
 #define HL_FW_BOOT_PTRS_BASE		0x80000
 #define HL_SPI_LEGACY_PREBOOT_OFF	0xC0000
